worldedit-core/src/main/java/com/fastasyncworldedit/core/math/BlockVectorSet.java

```diff
@@ -78,6 +78,49 @@ public boolean contains(int x, int y, int z) {
         return localMap != null && localMap.contains(x & 2047, ((y + 128) & 511) - 128, z & 2047);
     }
 
+    @Override
+    public void setOffset(final int x, final int z) {
+        // Do nothing
+    }
+
+    @Override
+    public void setOffset(final int x, final int y, final int z) {
+        // Do nothing
+    }
+
+    @Override
+    public boolean containsRadius(final int x, final int y, final int z, final int radius) {
+        if (radius <= 0) {
+            return contains(x, y, z);
+        }
+        // Quick corners check
+        if (!contains(x - radius, y, z - radius)) {
+            return false;
+        }
+        if (!contains(x + radius, y, z + radius)) {
+            return false;
+        }
+        if (!contains(x - radius, y, z + radius)) {
+            return false;
+        }
+        if (!contains(x + radius, y, z - radius)) {
+            return false;
+        }
+        // Slow but if someone wants to think of an elegant way then feel free to add it
+        for (int xx = -radius; xx <= radius; xx++) {
+            int rx = x + xx;
+            for (int yy = -radius; yy <= radius; yy++) {
+                int ry = y + yy;
+                for (int zz = -radius; zz <= radius; zz++) {
+                    if (contains(rx, ry, z + zz)) {
```

Comment:
So if any block intersects, it's considered to contain the radius? Could we call the method `intersectsWithRadius` or something like that then instead?

Reply:
I'm keeping the method name the same as the one that already existed in LocalBLockVectorSet